admin/src/main/ui/src/components/common/AxiosContext.js

```
@@ -0,0 +1,24 @@
+import {HttpError} from 'react-admin';
+
+const axios = host => {
+  const axios = Axios.create({
+    baseURL: host,
+    paramsSerializer: params => Qs.stringify(params, {allowDots: true, arrayFormat: 'repeat'}),
+  });
+
+  axios.interceptors.response.use(
+      response => response,
+      error => {
+        if (error.response && error.response.data && error.response.data.location) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)